Footer

JorgeAMendoza opened the pull request:
Create footer that is displayed on the dashboard for all pages, style the footer and implement test for the component.

JorgeAMendoza committed:
set footer structure

modify next js config to inject the version number into the application
and access it directly from the footer. create HTML base of the footer.

JorgeAMendoza committed:
style footer for mobile

implement the footer and place it within the page container so it can be
rendered on all routes (without having to import in into each route).
Modified structure of page container so the footer would not sit
directly in the content container.

JorgeAMendoza committed:
Style footer for desktop view

create height for footer to calculate the min height of the content
container, adjust spacing for footer and order of items in the element,
footer is placed within the main content but outside the container.

JorgeAMendoza committed:
refactor cypress test

inject application version into cypress runtime for test, create footer
test, and refactor test so only the targeted navigation element is
targeted.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-12)

JorgeAMendoza committed:
Update test and configuration

Create script to run Cypress in the command line, adjust Cypress config
to inject version number from the `package.json` file, fix test that
caused errors.
